README.md

```diff
@@ -255,22 +255,25 @@ There are a few different RDD's all named `Neo4jXxxRDD`
 
 ## DataFrames
 
-* `Neo4jDataFrame`, a SparkSQL `DataFrame` that you construct either with explicit type information about result names and types
-* or inferred from the first result-row
-* Neo4jDataFrame provides:
- 1.`mergeEdgeList(sc: SparkContext, dataFrame: DataFrame, source: (label,Seq[prop]), relationship: (type,Seq[prop]), target: (label,Seq[prop]))` to merge a DataFrame back into a Neo4j graph
+The `Neo4jDataFrame` is a SparkSQL `DataFrame` that you construct either with explicit type information about result
+names and types or inferred from the first result-row; it provides:
+ * `mergeEdgeList(sc: SparkContext, dataFrame: DataFrame, source: (label,Seq[prop]), relationship: (type,Seq[prop]), target: (label,Seq[prop]), partitions: Int = 1, unwindBatchSize: Int = 10000)` to merge a DataFrame back into a Neo4j graph
     * both nodes are merged by first property in sequence, all the others will be set on the entity
     * relationships are merged between the two nodes and all properties included in the sequence will be set on the relationship
     * property names from the sequence are used as column names for the data-frame, currently there is no name translation.<br/>
     *For example*: with a DataFrame containing a `screen` column describing the on-screen relationship between two actors, then to get a `screen` property on the relationship we'd need to supply a *relationship* tuple of `("ACTED_WITH",Seq("screen")`. If we had a `real` column in the DataFrame representing a real-life relationship as well, then we would need to add that property into the `Seq` too, e.g. `("ACTED_WITH",Seq("screen", "real"))` and so on.
     * the result are sent in batches of 10000 to the graph
-    * optional `renamedColumns` parameter - can be used to create a relationship between nodes having the same properties.<br/>
+    * optional `renamedColumns` parameter - can be used to create a relationship between nodes having the same properties.
+    * the `partitions` parameter defines the number of partitions of the dataframe will be repartitioned into. It defines the level of the parallelism, so consider that it could lead to transaction locks if the same node gets managed concurrently;
+    * the `unwindBatchSize` parameter defines for each partition the batch size sent to Neo4j.<br/>
  *For example*: `(keanu:Person {name: 'Keanu'})-[:ACTED_WITH]->(Laurence:Person {name: "Laurence"})` requires a DataFrame with 2 `name` columns which is not possible.
```

Comment:
the example is for renamedColums, need to keep it up there.

Reply:
done